### PR TITLE
refactor: change socket event

### PR DESCRIPTION
**Pull request by DmitriyHnatiuk**

### CODE REVIEW

File: src/pages/AdminLobby/index.tsx [hunk 1]
```diff
@@ -24,46 +24,50 @@ const { HOME } = ways;
...
-		}
-		const obj = { users: rooms.users };
+			return dispatch(setUserDataActionCreation({ users: rooms.users }));
+		});
 
-		return dispatch(setUserDataActionCreation(obj));
-	});
+		// socket.on('event://message', (ms, obj) => {
+		// 	console.log(ms, obj);
+		// });
```
Comment (Polina04) [inline src/pages/AdminLobby/index.tsx:64]: are you sure you need this?

File: src/pages/TeamMembers/index.tsx [hunk 2]
```diff
@@ -24,47 +24,46 @@ const TeamMembers: React.FC = () => {
...
+			}
+			return dispatch(setUserDataActionCreation({ users: rooms.users }));
+		});
 
-	socket.on('event://message', (ms, obj) => {
-		console.log(ms, obj);
-	});
+		// socket.on('event://message', (ms, obj) => {
+		// 	console.log(ms, obj);
+		// });
```
Comment (Polina04) [inline src/pages/TeamMembers/index.tsx:60]: are you sure you need it?

**Review by Polina04 — APPROVED** [review]

File: src/pages/AdminLobby/index.tsx (hunk 1 above)
Comment (DmitriyHnatiuk) [reply src/pages/AdminLobby/index.tsx:64]: yes, this part is not fully implemented yet

File: src/pages/TeamMembers/index.tsx (hunk 2 above)
Comment (DmitriyHnatiuk) [reply src/pages/TeamMembers/index.tsx:60]: if you ask me why it commits? I just added AnyEvent

Comment (DmitriyHnatiuk) [reply src/pages/TeamMembers/index.tsx:60]: but this part is implemented over time. maybe tomorrow, I don't know. this is for the chat component.

**Review by SyleimanovSergey — APPROVED** [review]

**Pull request merged**